cmd/query/app/handler_options.go

```diff
@@ -61,10 +61,10 @@ func (handlerOptions) QueryLookbackDuration(queryLookbackDuration time.Duration)
 	}
 }
 
-// Tracer creates a HandlerOption that initializes OpenTracing tracer
+// Tracer creates a HandlerOption that initializes Opentelemetry tracer
 func (handlerOptions) Tracer(tracer *jtracer.JTracer) HandlerOption {
 	return func(apiHandler *APIHandler) {
-		apiHandler.tracer = tracer
+		apiHandler.tracer.OTEL = tracer.OTEL
```

Comment:
why this change?

Reply:
I did this to workaround if it was desirable.